site/app/views/submission/HomeworkView.php

```diff
@@ -57,10 +57,32 @@ public function showGradeable($gradeable, $days_late) {
         $current_version_number = $gradeable->getCurrentVersionNumber();
         $return = <<<HTML
 <script type="text/javascript" src="{$this->core->getConfig()->getBaseUrl()}js/drag-and-drop.js"></script>
+<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
```

Comment:
We already import both jquery and jquery-ui (https://github.com/Submitty/Submitty/blob/master/site/app/views/GlobalView.php#L58-L59), so not sure why you're doing this here?